pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -254,6 +262,15 @@ func fitsRequest(podRequest *preFilterState, nodeInfo *schedulernodeinfo.NodeInf
 	return insufficientResources
 }
 
+func shouldIgnoreResource(rName string, groups sets.String) bool {
+	for group := range groups {
+		if strings.HasPrefix(rName, group) {
```

Comment:
can you look up in the apimachinery if there is something we can reuse? A prefix is too flexible. I prefer we adhere to some kind of regular expression.

Also, move `ignoredExtendedResources.Has` inside this function as well.

Reply:
ok, we can support `ignoredResourceGroups` as a list of the regular expressions.